scipy/special/add_newdocs.py

```diff
@@ -3071,10 +3078,15 @@ def add_newdoc(place, name, doc):
          valid for b,a,c,(b-a) != integer and (c-a),(c-b) != negative integer
 
       z >= 1 is rejected (unless special cases are present)
-    
+
+      a,b,c or z can take up complex values too.
```

Comment:
Also what can or cannot take complex values should be specified in the parameters section.